syntax = "proto3";
package user_service.proto;

import "cyberpay_utils/user/proto/user.proto";

service UserService {
  rpc AuthUser(AuthUserRequest) returns (AuthUserResponse) {}
  rpc GetUserById(GetUserByIdRequest) returns (GetUserByIdResponse) {}
}

// Requests
message AuthUserRequest {
  string login = 1;
  string password = 2;
}
message GetUserByIdRequest { uint64 id = 1; }

// Responses
message AuthUserResponse { User user = 1; }
message GetUserByIdResponse { User user = 1; }
